src/types/rooms.ts

```diff
@@ -19,6 +19,7 @@ export interface Room {
   width_percent: number;
   height_percent: number;
   isEnabled: boolean;
+  isLabeled?: boolean;
```

Comment:
I'm not a fan of this naming.. Our default is that labels will be shown as normal, so this exception should be named as changing that.

```suggestion
  isLabelHidden?: boolean;
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1632/commits/9e679604114d01a3121e1d3d42ac06b60632cc4a